BDArmory/Control/BDModulePilotAI.cs

```diff
@@ -2152,7 +2156,7 @@ void FlyToTargetVessel(FlightCtrlState s, Vessel v)
                 if (weaponManager.staleTarget) //lost track of target, but know it's in general area, simulate location estimate precision decay over time
                 {
                     if (staleTargetVelocity == Vector3.zero) staleTargetVelocity = v.Velocity(); //if lost target, follow last known velocity vector
-                    target += (staleTargetVelocity * weaponManager.detectedTargetTimeout) + (staleTargetPosition * weaponManager.detectedTargetTimeout);
+                    target += staleTargetVelocity + staleTargetPosition * weaponManager.detectedTargetTimeout;
```

Comment:
I think you want to swap `staleTargetVelocity` and `staleTargetPosition` here.

Reply:
So I do.